Simplenote/URLComponents.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension URLComponents {
+    static func simplenoteURLComponents(with host: String? = nil) -> URLComponents? {
+        var components = URLComponents(string: .simplenotePath())
+
+        if let host = host {
+            components?.host = host
+        }
```

Comment:
The host is declared optional, so we can directly set the host `var host: String? { get set }`